python/ray/train/sklearn/sklearn_predictor.py

```diff
@@ -23,16 +23,26 @@ class SklearnPredictor(Predictor):
             predictions.
         preprocessor: A preprocessor used to transform data batches prior
             to prediction.
+        use_gpu: If prediction is done on GPU where model will be moved to
+            GPU device upon instantiation. *Added only for consistent interface
+            with other predictors.*
     """
 
     def __init__(
-        self, estimator: BaseEstimator, preprocessor: Optional["Preprocessor"] = None
+        self,
+        estimator: BaseEstimator,
+        preprocessor: Optional["Preprocessor"] = None,
+        use_gpu: bool = False,
```

Comment:
this is to keep it consistent with other predictors, otherwise `test_sklearn_predictor` will fail as `BatchPredictor` will try to instantiate it with `use_gpu` flag. 

Alternatively we can choose to not set `use_gpu` flag in `BatchPredictor` tho, in this commit i made it always explicit for all predictor classes, but open to changes.